.github/actions/setup-common/action.yml

```diff
@@ -10,7 +10,7 @@ inputs:
   codec-rav1e:
     description: 'Can take the values: OFF, LOCAL, SYSTEM'
     default: 'OFF'
-  recent-cmake:
+  oldest-cmake:
```

Comment:
Optional: It would be good to document that by "oldest" we mean the minimum required version (i.e., the version specified in the `cmake_minimum_required()` call in CMakeLists.txt).

Reply:
done